docs/pages/2020_Intro_Week/sections/basics.rst

```diff
@@ -3,3 +3,961 @@
+typed in the ``verdi shell``, the latter will be marked in this document by a
+green background, like:
+
+.. code:: python
+
+    load_node(100) # A python verdi shell command
+
+while command-line instructions in ``bash`` to be typed into a terminal will
+be written with a blue background:
+
+.. code:: bash
+
+    verdi process list
+
+Alternatively, to avoid changing terminal, you can execute ``bash`` commands
+within the ``verdi shell`` or the notebook by adding an exclamation mark before
+the command itself:
+
+.. code:: ipython
+
+    !verdi process list
+
+.. _loadnode:
+
+Loading a node
+--------------
+
+Most AiiDA objects are represented by nodes, identified in the database by its
+``PK`` (an integer). You can access a node using the following command
+in the shell:
+
+.. code:: python
+
+    node = load_node(PK)
+
+Load a node using the ``PK`` of one of the calculations visible in the graph you
+displayed in the previous section of the tutorial. Then get the energy of the
+calculation with the command
+
+.. code:: python
+
+    node.res.energy
+
+You can also type
+
+.. code:: python
+
+    node.res.
+
+and then press ``TAB`` to see all the available output results of the
+calculation.
+
+Loading specific kinds of nodes
+-------------------------------
+
+Pseudopotentials
+~~~~~~~~~~~~~~~~
+
+From the graph you generated in  section :ref:`2020_virtual_aiidagraph`,
+find the ``PK`` of the pseudopotential file (LDA). Load it and
+show what elements it corresponds to by typing:
+
+.. code:: python
+
+    upf = load_node(PK)
+    upf.element
+
+All methods of ``UpfData`` are accessible by typing ``upf.`` and then pressing ``TAB``.
+
+k-points
+~~~~~~~~
+
+A set of k-points in the Brillouin zone is represented by an instance of the
+``KpointsData`` class. Choose one from the graph of
+produced in section :ref:`2020_virtual_aiidagraph`,
+load it as ``kpoints`` and inspect its content:
+
+.. code:: python
+
+    kpoints.get_kpoints_mesh()
+
+Then get the full (explicit) list of k-points belonging to this mesh using
+
+.. code:: python
+
+    kpoints.get_kpoints_mesh(print_list=True)
+
+If this throws an ``AttributeError``, it means that the kpoints instance does not represent a regular mesh but rather a list of k-points defined by their crystal coordinates (typically used when plotting a band structure).
+In this case, get the list of k-points coordinates using
+
+.. code:: python
+
+    kpoints.get_kpoints()
+
+Conversely, if the `KpointsData` node `does` actually represent a mesh, this method is the one, that when called, will throw an ``AttributeError``.
+
+If you prefer Cartesian (rather than crystal) coordinates, type
+
+.. code:: python
+
+    kpoints.get_kpoints(cartesian=True)
+
+For later use in this tutorial, let us try now to create a kpoints instance,
+to describe a regular (2 x 2 x 2) mesh of k-points, centered at the Gamma
+point (i.e. without offset). This can be done with the following commands:
```

Comment:
```suggestion
For later use in this tutorial, let us try now to create a kpoints instance, to describe a regular (2 x 2 x 2) mesh of k-points, centered at the Gamma point (i.e. without offset).
This can be done with the following commands:
```